src/DynamoCoreWpf/ViewModels/PackageManager/PackageItemRootViewModel.cs

```diff
@@ -29,6 +48,9 @@ public PackageItemRootViewModel(PackageAssembly assembly)
             this.Height = 32;
             this.DependencyType = DependencyType.Assembly;
             this.Assembly = assembly;
+            this.DisplayName = assembly.Name;
+            this.FilePath = assembly.Assembly.Location;
+            this.TargetPath = assembly.Assembly.Location;
```

Comment:
@QilongTang was this ever finished, it looks like TargetPath is the same as FilePath, I thought target path was supposed to allow the user to see where the file would end up? (ie in /bin, /extra etc.

Reply:
I think @OliverEGreen raised that question but we did not answer in time. Thank you for catching this. 

Here are comments from him:
```
Note: I am unsure what the Target Path is meant to show users. This dialog does not give users the option to select an output directory (which is what I assumed the Target Path would show). For now, it simply mirrors the File Path value.

I am also unable to read the File Path of .dyf files from the CustomNodeDefinition object. However, if there's a way to do this, please let me know and I will update.
```
@reddyashish you and I should take a look at the desired behavior for this field

Reply:
yeah, let's discuss this in the standup.

Reply:
After team discussion, we decided to hide this `target path` new functionality since it's missing key UI elements to set it like @OliverEGreen mentioned in the original PR. By default all files added in the dialog will simply go to `bin` folder, without a way to set the target path, the meaning is limited. @OliverEGreen Would you help us hide the UI display for now?